src/recommendationservice/metrics.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+# from gc import callbacks
+# from typing import Iterable
+# import psutil
+
+
+def init_metrics(meter):
+
+    # Requests counter
+    list_recommendations_request_counter = meter.create_counter(
```

Comment:
I think the metric `app_recommendations_request_counter` should also be replaced by instrumentation library. Since rpc server already has mature [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/rpc-metrics.md). 

you can get the total requests amount from `rpc_server_duration_count`. 

But I saw that maybe [python grpc instrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/__init__.py) library may not support meter, I think you can first contribute to that. =D